scripts/o3de/ExportScripts/export_source_ios_xcode.py

```diff
@@ -44,51 +64,81 @@ def export_ios_xcode_project(ctx: exp.O3DEScriptExportContext,
         logger = logging.getLogger()
         logger.setLevel(logging.ERROR)
 
-    tools_build_folder_str = str(tools_build_folder)
-    ios_build_folder_str = str(ios_build_folder)
+    default_base_path =  ctx.project_path
+
+    if not tools_build_path:
+        tools_build_path = default_base_path / 'build/tools'
+    elif not tools_build_path.is_absolute():
+        tools_build_path = default_base_path / tools_build_path
+
+    tools_build_path_str = str(tools_build_path)
+    ios_build_folder_str = str(target_ios_project_path)
+
+    # Resolve (if possible) and validate any provided seedlist files
+    validated_seedslist_paths = exp.validate_project_artifact_paths(project_path=ctx.project_path,
+                                                                    artifact_paths=seedlist_paths)
+    
+    # Preprocess the seed file paths in case there are any wildcard patterns to use
+    preprocessed_seedfile_paths = exp.preprocess_seed_path_list(project_path=ctx.project_path,
+                                                                paths=seedfile_paths)
+    
+    eutil.process_level_names(ctx, preprocessed_seedfile_paths, level_names, ASSET_PLATFORM)
 
     # Optionally build the toolchain needed to process the assets
+    # note: For now, I am keeping the tool building logic restricted on iOS, b/c atm it's unreliable for most configurations besides this one
     if should_build_tools:
-        exp.process_command(["cmake", "-B", tools_build_folder_str, '-G', "Xcode"], cwd=ctx.project_path)
+        exp.process_command(["cmake", "-B", tools_build_path_str, '-G', "Xcode"], cwd=ctx.project_path)
```

Comment:
is there a reason not to call `eutil.handle_tools()`?  We don't have an SDK yet for Mac, but I was wondering if we should add the `-#LY_DISABLE_TEST_MODULES=1` here to build faster.